docs/configuring/enabling-nio.html

```diff
@@ -0,0 +1,127 @@
+        </ul>
+        <li class="sidebar-item collapsible">Web Application</li>
+        <ul class="nested-sidebar-list content">
+          <li class="nested-sidebar-item"><a href="../templating/templating.html">What are Web Applications?</a></li>
+          <li class="nested-sidebar-item"><a href="../templating/static-file-server.html">Static File Server</a></li>
+          <li class="nested-sidebar-item"><a href="../templating/stencil.html">Stencil</a></li>
+          <li class="nested-sidebar-item"><a href="../templating/markdown.html">Markdown</a></li>
+        </ul>
+        <li class="sidebar-item collapsible">Deploying</li>
+        <ul class="nested-sidebar-list content" style="max-height: 400px;">
+          <li class="nested-sidebar-item"><a href="../deploying/monitoring.html">Monitoring</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/ssl.html">Enabling SSL/TLS</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/docker.html">Docker</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/kubernetes.html">Kubernetes</a></li>
+          <li class="nested-sidebar-item active"><a href="../deploying/cloud-foundry.html">Cloud Foundry</a></li>
+          <li class="nested-sidebar-item active"><a href="../configuring/enabling-nio.html">Enabling SwiftNIO</a></li>
+        </ul>
+      </ul>
+    </aside>
+    <div id="burgerIcon" class="burger-icon" onclick="showSidebar()">
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+    </div>
+    <div class="docs-item-2 search-container">
+    </div>
+    <nav class="docs-item-3 docs-nav">
+      <button id="api-button" class="apiref-button" type="button" name="button" onclick="window.open('https://ibm-swift.github.io/Kitura/')">API Reference</button>
+      <a class="nav-item" target="_blank" href="http://slack.kitura.io/">Need help?</a>
+      <a class="nav-item" target="_blank" href="https://github.com/IBM-Swift/kitura.io/issues">Found an issue?</a>
+    </nav>
+    <div id="doc-container" class="docs-item-4 docs-window">
+      <main>
+        <h1 class="heading-1">Enabling SwiftNIO</h1>
+        <p>In February 2018, Apple announced <a href="https://github.com/apple/swift-nio">SwiftNIO</a> — a new cross-platform, asynchronous, event-driven networking framework for building servers and clients in Swift. SwiftNIO has the potential to provide all the low-level socket-handling and HTTP protocol support in Kitura, and hence could replace the current <a href="https://github.com/IBM-Swift/Kitura-net">Kitura-net</a> and <a href="https://github.com/IBM-Swift/BlueSocket">BlueSocket</a> libraries which underpin Kitura today. The Kitura team released <a href="https://github.com/IBM-Swift/Kitura-NIO">Kitura-NIO</a>, a Kitura interface to SwiftNIO in May 2018.</p>
```

Comment:
Replace this paragraph with "SwiftNIO is a cross-platform, asynchronous, event-driven networking framework for building network servers and clients in Swift. Since Kitura 2.5 you have the option of running your Kitura application using SwiftNIO instead of the Kitura-net and BlueSocket libraries which Kitura has traditionally been based upon. This support is enabled through the Kitura-NIO package."